packages/gunshi/src/definition.ts

```
@@ -37,14 +72,41 @@
   loader: CommandLoader<A>,
   definition?: Command<A>
-): LazyCommand<A> {
+): LazyCommand<A>
+
+/**
+ * Define a {@link LazyCommand | lazy command} with extension support
+ * @param loader A {@link CommandLoader | command loader}
+ * @param definition An {@link ExtendedCommand | extended command} definition with extensions
+ * @returns A {@link LazyCommand | lazy command} loader with extension support
+ */
+export function lazy<A extends Args = Args, E extends Record<string, ContextExtension> = {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)